tests/test_window_streams.py

```diff
@@ -129,8 +125,11 @@ def test_error_handling():
         # 1. Return empty/zero data
         # 2. Skip the error and continue on next valid data
         # 3. Or maintain last valid state
-        window2 = next(generator)
-        assert window2 is not None  # Shouldn't crash
+        try:
```

Comment:
This test doesn't make much sense. The assertions are mutually exclusive in execution, and I don't know which one is supposed to run

Reply:
That's the 5-minutes-before-party code XD Will clean up the tests and make them all clear, informative and relevant.